### DISCUSSION TITLE
Hardware keyboard mode

### DISCUSSION THREAD

**Discussion by anonimo82** [coder/code-server#5617] [Q&A] [answered]
Hello everyone.
Ok, this is one I stolen from VH-Editor, but I wonder if it's possible in Code Server too.
Simply put, is it there any way to force keyboard mode to hardware?

In VH-Editor this improves menu navigation and general UX.

Thanks for any help.

**Comment by jsjoeio** [comment]
Can you elaborate? What I'm hearing is you're using code-server on a device like an iPad or mobile phone and it's displaying the keyboard which is cluttering the screen? And you want to disable it when a keyboard is attached?

**Comment by anonimo82** [comment]
To correctly elaborate for this, I'm going to ask VH-Editor's developer how did they implemented this feature and get back here ASAP.

Edit: I asked for help here: https://github.com/vhqtvn/VHEditor-Android/issues/197

**Comment by anonimo82** [comment, marked as answer]
I got a reply, please check the link.
Long story short, ain't possible.
